### DISCUSSION TITLE
Ability to merge claims

### DISCUSSION THREAD

**Discussion by YanisBft** [GriefPrevention/GriefPrevention#2032] [Ideas]
Imagine a player made multiple claims near each other, and they have to edit them. In that case, having the ability to merge all the claims in an area would be very helpful.

How would it work?
1. Like when you claim, you select the two corners of the area
2. Then, you execute /mergeclaims
3. If you are the owner of all the claims inside the area, the merge is successful

What do you think?

**Comment by RoboMWM** [comment]
I've had an idea like this before, not sure of best way to implement. When I ran a server, issues would occur when people make multiple claims in an area and not realize this when doing things like /trust for others. Automatic merging would be cool, but probably not easily possible nor make sense unless both claims are adjacent and still would form a rectangle.

Given the complexity, this would very likely be an addon-only candidate.
